test/support/test_helpers.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Twitter.TestHelpers do
+  alias Twitter.{
+    Accounts
+  }
+
+  alias Twitter.{
+    Twits
+  }
+
+  def user_fixture(attrs \\ %{}) do
+    {:ok, user} =
+      attrs
+      |> Enum.into(%{
+        name: "Some User",
+        username: "user#{System.unique_integer([:positive])}",
+        password: attrs[:password] || "supersecret",
+        email: "email"
+      })
+      |> Accounts.create_user()
+
+    user
+  end
+
+  def post_fixture(user, attrs \\ %{body: "some body"}) do
```

Comment:
[nit] I think it is best to keep the `attrs \\ %{}` / `Enum.into(%{  ...default map })`